trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/IMemoryMetadata.java

```diff
@@ -0,0 +1,23 @@
+package de.uni_freiburg.informatik.ultimate.cdt.translation.implementation.base.chandler;
+
+import java.util.List;
+
+import de.uni_freiburg.informatik.ultimate.boogie.ast.Declaration;
+
+public interface IMemoryMetadata {
+	/**
+	 * Constructs the required metadata for the selected addressing mode.
+	 *
+	 * @param requiredFeatures
+	 *            The Features that are currently needed for the program to be verified.
+	 * @return The metadata declarations.
+	 */
+	List<Declaration> constructMetaData(RequiredMemoryModelFeatures requiredFeatures);
+
+	/**
+	 * Returns a list of metadata declarations needed for the memory model infrastructure.
+	 *
+	 * @return The declarations.
+	 */
+	List<MemoryModelDeclarations> metaDataDeclarations();
```

Comment:
Add prefix `get` to interface method name.

```suggestion
	List<MemoryModelDeclarations> getMetaDataDeclarations();
```

Reply:
See [3cb57aa](https://github.com/ultimate-pa/ultimate/pull/756/commits/3cb57aac1b47ed100add1c3c560b8ae9666ca74c)